tools/pythonpkg/tests/fast/test_parquet.py

```diff
@@ -110,5 +112,29 @@ def test_from_parquet_binary_as_string_default_conn(self,duckdb_cursor):
         res = rel.execute().fetchall()
         assert res[0] == ('foo',)
 
+    def test_from_parquet_union_by_name(self, duckdb_cursor):
+        conn = duckdb.connect()
```

Comment:
Can you switch this to using the pytest temp path instead?

Reply:
Fixed